sequencing/vcf_qc/dxapp.json

```diff
@@ -102,62 +175,39 @@
       "label": "Additional Hard Filters",
       "class": "string",
       "optional": true
-    },
-    {
-      "group": "Build Options",
-      "name": "build_version",
-      "label": "NCBI Build Version",
-      "class": "string",
-      "choices": ["rgc_b38","gatk_b38","b37_decoy","hg38chr.fa"],
-      "default": "rgc_b38",
-      "optional": false
     }
   ],
   "outputSpec": [
     {
-      "name": "vcf_out",
-      "label": "Filtered VCF file",
+      "name": "out_vcfgz",
+      "label": "Filtered VCF files",
       "class": "array:file",
-      "patterns": [
-        "*.vcf.gz"
-      ]
+      "patterns": ["*.vcf.gz"]
     },
     {
-      "name": "vcfidx_out",
-      "label": "Filtered VCF Index file",
+      "name": "out_vcfidx",
```

Comment:
"out_vcfidx" -> "out_variants_vcfgztbis"
Also need to update in the script accordingly.